package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "node-firefox-forward-ports",
+  "version": "1.0.0",
+  "description": "Find attached FirefoxOS devices using ADB",
```

Comment:
dear native English speakers, is *attached* the right word? would *connected* or *plugged in* be better? I feel like an attachment is more superficial?